roles/update-host/tasks/wait-for-host.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: "Ensure the ansible_ssh_host is defined"
```

Comment:
something doesn't seem right about this, for 2 reasons:
1) setting `ansible_ssh_host` to `hostvars[inventory_hostname]`, then the lines below will check if `ansible_host` or `ansible_ssh_host` is a key in that dictionary - seems like a catch-22
2) Since we are already setting `update_host` below, we should just improve that one set_fact instead of having 2 set_fact handling similar tasks.

Reply:
You could check it now? `update_host` is eliminated, and I've added a default sequence for which host to reboot. Also I've added the 2.7 reboot host module when necessary. Changed the import to include vars so that everything would work. I think the changes on the particular file are much more elegant than setting a bunch of unnecessary facts.